ckanext/recombinant/templates/recombinant/snippets/activity_stream.html

```diff
@@ -0,0 +1,3 @@
+<a id="activity-stream-link" href="{{ h.url_for(controller='package', action='activity', id=pkg.id) }}">
+    {{ _('Activity Stream') }}
+</a>
```

Comment:
How about a block instead of a snippet? snippets add quite a bit of delay to rendering time and it doesn't seem worthwhile in this case